lametro/bills.py

```diff
@@ -23,15 +23,35 @@ class LametroBillScraper(LegistarAPIBillScraper, Scraper):
 
     def __init__(self, *args, **kwargs):
         '''
-        Initialize the Bill scraper with default param values,
-        and set the `scrape_restricted` property to True. 
-        Together, they enable the scraping of private bills, i.e., 
-        bills with 'MatterRestrictViewViaWeb' set as True.
+        Metro scrapes private (or restricted) bills. 
+        Private bills have 'MatterRestrictViewViaWeb' set as True
+        and/or a MatterStatusName of 'Draft' and/or do not appear in the 
+        Legistar web interface.
+
+        The following properties enable scraping private bills:
+        :params - URL params that include the secret Metro API Token
+        
+        :scrape_restricted - a boolean used in `python-legistar-scraper`: it
+        indicates that the scrape should continue, even if the bill does not exist in
+        the Legistar web interface
+        
+        :start_date_private_scrape - a timestamp that indicates when to start scraping 
+        private bills. The scraper can safely skip bills from early legislative sessions, 
+        because those bills will remain private.
         '''
         super().__init__(*args, **kwargs)
 
         self.params = {'Token': TOKEN}
-        self.scrape_restricted = False
+        self.scrape_restricted = True
```

Comment:
if these are always the same value, i.e., they can't be set by passing in args or kwargs when initializing this class and aren't altered by any of the methods on this class, it seems more straightforward to me to set them as [class attributes, rather than instance attributes](https://www.toptal.com/python/python-class-attributes-an-overly-thorough-guide).

Reply:
(if they should be able to be set when initializing an instance, then we should set them from kwargs like `kwargs.get('scrape_restricted', True)` or provide explicit kwargs and default values in the init method definition.)

Reply:
ah, yes. the second suggestion makes sense, but for a sensical implementation, we'd to adjust further upstream: https://github.com/opencivicdata/python-legistar-scraper/blob/master/legistar/bills.py#L225 would you like to see that?

Reply:
eh, i can think of better ways to use time, to be honest.